sound/soc/sof/intel/hda-pcm.c

```diff
@@ -213,10 +213,10 @@ snd_pcm_uframes_t hda_dsp_pcm_pointer(struct snd_sof_dev *sdev,
 }
 
 int hda_dsp_pcm_open(struct snd_sof_dev *sdev,
-		     struct snd_pcm_substream *substream)
```

Comment:
@keyonjie which "SOF specific information" do you need easy access to. Neither the patch nor the commit message make it clear